src/Functions/randCanonical.cpp

```diff
@@ -47,13 +47,20 @@ class FunctionCanonicalRand : public FunctionRandomImpl<CanonicalRandImpl, Float
 
 REGISTER_FUNCTION(CanonicalRand)
 {
-    factory.registerFunction<FunctionCanonicalRand>(FunctionDocumentation{
-        .description=R"(
-The function generates pseudo random results with independent and identically distributed uniformly distributed values in [0, 1).
-Non-deterministic. Return type is Float64.
-        )",
-        .examples{{"randCanonical", "SELECT randCanonical()", ""}},
-        .category = FunctionDocumentation::Category::RandomNumber});
+    FunctionDocumentation::Description description = R"(
+Returns a random `Float64` number.
+    )";
+    FunctionDocumentation::Syntax syntax = "randCanonical()";
+    FunctionDocumentation::Arguments arguments = {};
+    FunctionDocumentation::ReturnedValue returned_value = {"Returns a value between `0` (inclusive) and `1` (exclusive).", {"Float64"}};
```

Comment:
Let's simplify: `A random number of type `Float64`.` (the range is now already part of the description).